pyatlan/model/enums.py

```diff
@@ -1721,6 +1721,76 @@ class TypeName(str, Enum):
     ARRAY_STRING = "array<string>"
 
 
+class AtlanWorkflowPhase(str, Enum):
+    SUCCESS = "Succeeded"
+    RUNNING = "Running"
+    FAILED = "Failed"
+    ERROR = "Error"
+    PENDING = "Pending"
+
+
+class ChildScoreMode(str, Enum):
+    NONE = "none"
+    AVG = "avg"
+    SUM = "sum"
+    MAX = "max"
+    MIN = "min"
+
+
+class WorkflowPackage(str, Enum):
+    ATLAN_AIRFLOW = "atlan-airflow"
```

Comment:
One other — every single one of these starts with `ATLAN_`, which I think is a bit redundant. Can we drop that prefix from the enum?

So we get:

- `AIRFLOW`
- `ATHENA`
- etc